graviti/utility/file.py

```diff
@@ -6,14 +6,65 @@
 """Graviti file related class."""
 
 from functools import partial
+from typing import Callable, Optional, Union
+
+from _io import BufferedReader
+from tensorbay.utility.repr import ReprMixin
+
+from graviti.exception import ResponseError
+from graviti.utility.requests import UserResponse, config, get_session
+
+
+class URL:
```

Comment:
seems we do not need `URL` class for now, can we just replace it with `str`

Reply:
for now, we need `URL` only in file.py, by `READ_ONLY_URL`, which is needed by `File`  class

Reply:
just use `url:str`